app/assets/stylesheets/components/firm_list/_firm_list.scss

```diff
@@ -23,3 +23,19 @@
 .firm-list__item__name {
   font-weight: bold;
 }
+
+.firm-list__filter {
+  font-size: $baseline-unit*2;
+  color: $color-firm-list-border-grey;
+  margin-top: 44px;
+  margin-left: 0.5em;
+  padding: 0.5em;
+  -webkit-border-radius: 15px;
+  -moz-border-radius: 15px;
+  border-radius: 15px;
+  border: 2px solid  $color-firm-list-border-grey;
+}
+
+#firm-list h2 {
```

Comment:
This goes against our CSS convention, can this be given a proper BEM name or use a layout?